packages/algob/src/lib/algo-operator.ts

```diff
@@ -60,4 +70,56 @@ export class AlgoOperatorImpl implements AlgoOperator {
       confirmedRound: txConfirmation[confirmedRound]
     };
   }
+
+  async deployASC (programb64: string, scParams: object, flags: ASCDeploymentFlags
+  ): Promise<ASCInfo> {
+    const program = new Uint8Array(Buffer.from(programb64, "base64"));
+    const encoder = new TextEncoder();
```

Comment:
are you sure we should use `TextEncoder` ( thought we need to it as base64, but I'm not sure).

Reply:
this is for the "note"  parameter

Reply:
program is need as base64, but if a user gives notes then we need to convert it first then only we can pass it to transaction function